composer.json

```diff
@@ -114,6 +114,7 @@
         "drupal/y_donate": "^2.0",
         "grt107/grt-youtube-popup": "^1.0",
         "guzzlehttp/guzzle": "^6 || ^7",
+        "league/csv": "^9.11.0",
```

Comment:
Can we also add here condition about != specific, non-working version? @revathiymca

Reply:
Updated the same @podarok